backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -48,6 +52,62 @@
 class PromptStudioHelper:
     """Helper class for Custom tool operations."""
 
+    @staticmethod
+    def create_default_profile_manager(user: User, tool_id: uuid) -> None:
+        """Create a default profile manager for a given user and tool.
+
+        Args:
+            user (User): The user for whom the default profile manager is
+            created.
+            tool_id (uuid): The ID of the tool for which the default profile
+            manager is created.
+
+        Raises:
+            AdapterInstance.DoesNotExist: If no suitable adapter instance is
+            found for creating the default profile manager.
+
+        Returns:
+            None
+        """
+        try:
+            AdapterInstance.objects.get(
+                is_friction_less=True,
+                is_usable=True,
+                adapter_type=AdapterKeys.LLM,
+            )
```

Comment:
@johnyrahul why are we doing this?

Reply:
@chandrasekharan-zipstack 
For handling scenario where the customer has used up his free LLM tokens which was given during onboarding.
So there is no point in setting up an default LLM profile (with usage exhausted LLM) which cant be used further.

Reply:
@johnyrahul This should be ok. If upgrading plan when remaining token count reaches zero is the only natural option out, then always adding a default LLM profile would reduce one step for the user.

Reply:
Got it, so we'd catch the exception and continue. It wasn't immediately obvious to me - we could add it with a comment maybe @johnyrahul